src/column-layout/interfaces.ts

```diff
@@ -27,6 +27,14 @@ export interface ColumnLayoutProps extends BaseComponentProps {
    */
   disableGutters?: boolean;
 
+  /**
+   * Use together with `columns` to specify the desired minimum width for each column in pixels.
+   *
+   * The number of columns is determined based on the value of this property, the available space,
+   * and the maximum number of columns as defined by the `columns` property.
+   */
+  minColumnWidth?: number;
```

Comment:
When using columns with minColumnWidth it does not have limit for 4 columns. Shall we make it clearer in documentation?

Reply:
Good idea. Let me think about that.